src/developer/web-api/openapi.md

```diff
@@ -42,4 +42,22 @@ Available tags are:
 * `integration`
 * `login`
 * `query`
-* `management`
\ No newline at end of file
+* `management`
+
+All endpoints that generate a OpenAPI document support the following optional 
+request parameters:
+
+### `failOnNameClash`
+When set to `true`, two or more types of same simple (unqualified) name are considered clashing and the generation fails with an error. 
+
+When set `false` (default), name clashes are resolved by adding numbers to the simple name to make each of them unique.
+As a result the names are not predictable or stable. Merging with documentation based on name will be broken. 
```

Comment:
I don't understand

> Merging with documentation based on name will be broken. 
This option is meant as a preview feature. 

What do you mean by `merging`? Me generating the docs and committing them to a git repo where they are kept?

Reply:
No, I was referring to the markdown documents from which we extract the description texts. The keys will no longer line up and thus no description will be found.

Reply:
Ah got it 😁 Maybe this makes it more clear then

```suggestion
As a result the names are not predictable or stable. Merging simple names with their intended markdown documentation based on name will be broken. 
```